packages/charts/react-charting/src/components/DeclarativeChart/DeclarativeChart.tsx

```diff
@@ -93,12 +104,31 @@ export const DeclarativeChart: React.FunctionComponent<DeclarativeChartProps> =
     ...(activeLegends.length > 0 && { selectedLegend: activeLegends[0] }),
   };
 
+  const download = React.useCallback(
```

Comment:
>download [](http://example.com/codeflow?start=8&length=8)

rename to downloadAsImage

Reply:
Should I rename it to exportToImage, since we are using the term 'export' everywhere, and it just converts SVG to image data, which can be saved or downloaded by the user?

Reply:
done